### DISCUSSION TITLE
recipes/breakfast/costa-rican-baked-doughnuts/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#718] [Comments]
# recipes/breakfast/costa-rican-baked-doughnuts/

A collection of my recipes.

https://nicholaswilde.io/recipes/breakfast/costa-rican-baked-doughnuts/

**Comment by nicholaswilde** [comment]
- Add link to milk glaze. 1/2 cup sugar does half a batch.
- food process the corn meal to make finer.
- refrigerate the dough 1/2 hour.
- use 1/2 - 2/3 water. Creamed corn makes it more liquid.
